WcaOnRails/app/models/user.rb

```diff
@@ -293,6 +285,49 @@ def after_confirmation
     end
   end
 
+  def software_team?
+    team_member?('software')
+  end
+
+  def results_team?
+    team_member?('results')
+  end
+
+  def wrc_team?
+    team_member?('wrc')
+  end
+
+  def wdc_team?
+    team_member?('wdc')
+  end
+
+  def team_member?(team)
+    self.team_members.where(team_id: ( self.teams.find_by_friendly_id(team) ) ).each do |member|
```

Comment:
I think this would be cleaner if you used Ruby's `any?` method here instead. Maybe:

```ruby
self.team_members.where(team_id: ( self.teams.find_by_friendly_id(team) ) ).any?(&:current_member?)
```